rps.js

```diff
@@ -41,5 +32,18 @@ var compare = function(choice1, choice2){
   }
 }
 
-compare(userChoice, computerChoice);
-});
\ No newline at end of file
+$("#userChoice button").click(function(){
+  userChoice = $(this).val();
+  computerChoice = Math.random();
+  if (computerChoice < 0.34) {
+    computerChoice = "rock";
+  } else if(computerChoice <= 0.67) {
+    computerChoice = "paper";
+  } else {
+    computerChoice = "scissors";
+  } 
+  $("#computerChoice").html(computerChoice);
```

Comment:
This puts the value of computerChoice inside the div with the id "computerChoice".